### PR TITLE
Portf 1136 allow 3des cbc in dropbear ssh client

### PR DESCRIPTION
**Pull request by yba-siklu**
## Description
 * Build separate dropbear client and server programs instead on default multi-executable binary.
 * Create separate configuration files for dropbear server and client programs.
 * Add new configuration options to Buildroot dropbear package for legacy ciphers for dropbear client and server programs separately.
 * Update Buildroot defconfig to use legacy ciphers (including 3des-cbc) for dropbear client (dbclient)

## Tests
 * Tested on 8020 with "ssh -c help"

## References
JIRA: PORTF-1136

### CODE REVIEW

**Comment by alexandert12** [comment]
@yba-siklu , please, did you test it on EH-8010 ?
What is the output of the "ssh -c help" command ?
How did you test SSH client ? Do you have SSH server, which uses 3des-cbc cipher ?
How did you test SSH server ? Do you have SSH client, which uses 3des-cbc cipher ?

File: configs/siklu_80X0_defconfig [hunk 1]
```diff
@@ -18,7 +18,6 @@ BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
 BR2_TOOLCHAIN_EXTERNAL_CXX=y
 BR2_TOOLCHAIN_EXTERNAL_GDB_SERVER_COPY=y
 BR2_TARGET_GENERIC_ROOT_PASSWD="123456"
-BR2_PACKAGE_GESFTPSERVER=y
```
Comment (alexandert12) [inline configs/siklu_80X0_defconfig:21]: Why it is removed ?

**Comment by yba-siklu** [comment]
> @yba-siklu , please, did you test it on EH-8010 ? What is the output of the "ssh -c help" command ? How did you test SSH client ? Do you have SSH server, which uses 3des-cbc cipher ? How did you test SSH server ? Do you have SSH client, which uses 3des-cbc cipher ?

I did not test on 8010 as I do not have easy access to an 8010.  "ssh -c help" shows compile-in ciphers. I did not test yet against an actual server.

**Comment by alexandert12** [comment]
Showing compile-in ciphers does not prove that SSH with cipher 3des-cbc actually works.
I do not recommend to push these changes till the testing against an actual server passes successfully.

@yba-siklu , please, use setup #12, or #13, or #16 to test on EH-8010.

**Pull request closed without merging**